### PR TITLE
Add new endpoint for tsne metadata for tsne plot widget

### PR DESCRIPTION
**Pull request by haideriqbal**
Added new endpoint for tsne plot metadata to be used by tsne plot widget. Endpoint returns array of metadata.
Added new test cases for new metadata endpoint to check following:
- Valid Json for invalid metadata
- Valid Json for valid metada
- Valid Json for valid ExperimentAccession

Link to green build:

http://bar:8085/browse/ATLAS-ATLASWEBSCGRCI3-1

### CODE REVIEW

File: src/test/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentTSnePlotControllerWIT.java [hunk 1]
```diff
@@ -38,6 +42,8 @@
 @ContextConfiguration(classes = TestConfig.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class JsonExperimentTSnePlotControllerWIT {
+    private static final String URL = "/json/experiments/{experimentAccession}/metadata";
```
Comment (monicajianu) [inline src/test/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentTSnePlotControllerWIT.java]: Seeing as there are quite a few other tests in this class that use other URLs, could you change this to something a little less generic, e.g. `EXPERIMENT_METADATA_ENDPOINT_URL`?

**Review by monicajianu — COMMENTED** [review]
This looks much simpler and cleaner! Well done! Could you add one tiny additional test to retrieve metadata for an experiment that doesn't have any metadata (e.g. E-GEOD-99058), which should return an empty array? After that you'll be good to go!

File: src/test/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentTSnePlotControllerWIT.java (hunk 1 above)
Comment (haideriqbal) [reply src/test/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentTSnePlotControllerWIT.java]: Done!

**Review by alfonsomunozpomer — COMMENTED** [review]
Seems good.

**Pull request closed without merging**